cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -523,7 +523,7 @@ const clients = function (region, rootDnsRole) {
  */
 exports.certificateRequestHandler = async function (event, context) {
   var responseData = {};
-  var physicalResourceId;
+  var physicalResourceId = event.PhysicalResourceId;
```

Comment:
This changes the original behavior a little bit for "Update". Originally, it is possible that `physicalResourceId` is the logstreamName, now it's not because `event.PhysicalResourceId` is always given with `Update` and `Delete`. Would this matter?

Reply:
I think this is a bug fix so i would say it's ok.